aruco_detector_node:
  ros__parameters:
    aruco_dictionary: "DICT_5X5_250"